doc/user/layouts/shortcodes/network-security/privatelink-kafka.md

```diff
@@ -52,14 +52,34 @@ and retrieve the AWS principal needed to configure the AWS PrivateLink service.
 1. #### Create an AWS PrivateLink connection
      In Materialize, create an [AWS PrivateLink connection](/sql/create-connection/#aws-privatelink) that references the endpoint service that you created in the previous step.
 
-     ```mzsql
-    CREATE CONNECTION privatelink_svc TO AWS PRIVATELINK (
-        SERVICE NAME 'com.amazonaws.vpce.<region_id>.vpce-svc-<endpoint_service_id>',
-        AVAILABILITY ZONES ('use1-az1', 'use1-az2', 'use1-az3')
-    );
-    ```
+    **Availability Zone Selection**:
+
+    * For in-region connections, the AZs **must match** between the NLB and the consumer VPC.
+    * For cross-region connections, AZ alignment is not required. You can choose any AZs in your consumer VPC.
```

Comment:
Is this is because cross zone balancing is a pre-req of cross region connections?

Reply:
If I read their [docs](https://aws.amazon.com/blogs/networking-and-content-delivery/introducing-cross-region-connectivity-for-aws-privatelink/) correctly, this seems to be the case for providers (in our case customers):

> Providers must use a minimum of two AZs to anchor the [AWS Network Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html) (NLB) in the service region. Consumers can choose to create the Interface endpoint in as many AZs as required, although two or more AZs are recommended for high availability. This enables PrivateLink to dynamically weigh away traffic to a healthy AZ, transparent to the provider and consumer, in case of AZ impairment events in either region.